src/i18n/languages.ts

```diff
@@ -17,4 +17,13 @@ export default {
 	ru: 'Русский',
 } as const;
 
+if (import.meta.env.PUBLIC_ONLY_EN) {
+	// @ts-expect-error This conflicts with the const type above, but it's fine when building
```

Comment:
This feels weird, but I guess it’s better than `languages = import.meta.env.PUBLIC_ONLY_EN ? { ... } : { ... }`?

Reply:
I didn't want the types to be affected, but I can try the ternary to see if it builds fine

Reply:
I had to use a `Record<string, string>` to make `pnpm tsc` work. Using `as const` would make the types having `[key]?: string` (undefined-able), which makes typechecking fail. Happy to walk back to the previous hack too if you prefer to keep the `as const` type.